app/views/admin/categories/index.html.haml

```diff
@@ -7,6 +7,8 @@
 
   %div.my-1.border-radius-0
     = render_alerts_container
+    .rounded-json-button
+      = render_rounded_button_with_style('display: flex; justify-content: end; right: 168px; position: absolute; top: 31px;', "#{$REST_URL}/categories?display=all")
```

Comment:
all the links need to include the APIKEY otherwise they will not open (try to open them in private mode you will see)